tests/integration/graviton_test.py

```diff
@@ -961,22 +960,28 @@ def named_tuple_field_access(
         )
 
     lsig_pytealer = PyTealDryRunExecutor(named_tuple_field_access, Mode.Signature)
+    args = (False, b"1" * 32, (0, False), b"0" * 10, [True] * 4, 0)
 
-    TYPES_N_VALS: list[tuple[str, Any]] = [
-        ("bool", False),
-        ("address", b"1" * 32),
-        ("(uint64,bool)", (12, True)),
-        ("byte[10]", b"a" * 10),
-        ("bool[4]", [False] * 4),
-        ("uint64", 100),
-    ]
-
-    args: list[bytes] = [
-        ABIType.from_string(abi_t).encode(var) for abi_t, var in TYPES_N_VALS
-    ]
-
-    lsig_result = lsig_pytealer.dryrun(args)
-    assert lsig_result.passed()
+    inspector = lsig_pytealer.dryrun(args)
+
+    assert inspector.stack_top() == 1
+    assert inspector.passed()
+
+    # TYPES_N_VALS: list[tuple[str, Any]] = [
+    #     ("bool", False),
+    #     ("address", b"1" * 32),
+    #     ("(uint64,bool)", (12, True)),
+    #     ("byte[10]", b"a" * 10),
+    #     ("bool[4]", [False] * 4),
+    #     ("uint64", 100),
+    # ]
+
+    # args: list[bytes] = [
+    #     ABIType.from_string(abi_t).encode(var) for abi_t, var in TYPES_N_VALS
+    # ]
+
+    # lsig_result = lsig_pytealer.dryrun(args)
+    # assert lsig_result.passed()
```

Comment:
if graviton can do all the stuffs from python var to ABI encodings, that is great! I don't see any point we should keep my previous snippet of implementation here tho...

Reply:
ok, I'll update without

Reply:
updated

Reply:
👍